scripts/rebuild.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+PACKAGE_DIR=${PACKAGE_DIR:-./.cr-release-packages}
+CHARTS_DIR=${CHARTS_DIR:-charts}
+INDEX=${INDEX:-index.yaml}
+HELM_REPO=${HELM_REPO:-https://helm.rstudio.com}
+GITHUB_OWNER=${GITHUB_OWNER:-rstudio}
+GITHUB_REPO=${GITHUB_REPO:-helm}
+
+# Run this script from the root of the helm repo, e.g.
+# bash scripts/rebuild.sh
```

Comment:
Super nit but if we make this script executable it's more convenient imo to run `./scripts/rebuild.sh` (as we are explicitly using bash via line 1).